tests/test_kernels.py

```diff
@@ -0,0 +1,89 @@
+"""Kernel tests."""
+
+from typing import Any
+
+import numpy as np
+import torch
+from attrs import asdict, has
+from hypothesis import given
+
+from baybe.kernels.base import Kernel
+from baybe.kernels.composite import AdditiveKernel, ProductKernel, ScaleKernel
+
+from .hypothesis_strategies.kernels import kernels
+
+# TODO: Consider deprecating these attribute names to avoid inconsistencies
```

Comment:
Because we use a different attribute name than gpytorch, we now require this workaround. We can either
* keep it as is
* rename our attribute with a (rather simple) deprecation
* do the deprecation anytime in the future.

Preferences?

Reply:
Also do not have a strong opinion here. I think it makes no big difference whether we have `kernel` or `base_kernels`, so I'd go with whatever is the cleanest from a code perspective.

Reply:
OK, then I'll simply keep things as they are for now